integration/test/upgradetiller/upgrade_tiller_test.go

```diff
@@ -16,46 +16,11 @@ import (
 func TestUpgradeTiller(t *testing.T) {
 	ctx := context.Background()
 
-	var err error
-
 	currentTillerImage := "quay.io/giantswarm/tiller:v2.14.3"
-	outdatedTillerImage := "quay.io/giantswarm/tiller:v2.12.0"
 
 	labelSelector := "app=helm,name=tiller"
 	tillerNamespace := "giantswarm"
 
-	// Install tiller using current image.
```

Comment:
Deleted the first `EnsureTillerInstalled` usage in `UpgradeTillerTest` since it's not what we want to test but too error-prone.

Reply:
Why are we dropping the downgrade part of the test ?

Reply:
We used `EnsureCreated` to create the latest version of tiller pod and downgrade it, but it was too error-prone; sometimes `getPod` didn't get the pod right for various reasons. I think installing helm from the beginning of e2etest is better to fit in real cases.

Reply:
I'm fine with the change of approach. The original approach I used was to install helm via EnsureTillerDeploying using the installer and then update the deployment to reduce the version.

Looks like this is unreliable and installing helm via the CLI is arguably a better test.